pkgs/development/libraries/libyaml/default.nix

```diff
@@ -1,14 +1,14 @@
 { stdenv, fetchurl }:
 let
   # 0.2.1 broke the tests of pythonPackages.pyyaml 3.13
```

Comment:
This comment should be removed as part of the update.

Reply:
Well, yes, but first the issue need to be solved :-)